plugins/module_utils/oracle/oci_utils.py

```diff
@@ -561,7 +561,7 @@ def are_lists_equal(s, t):
     if s is None and t is None:
         return True
 
-    if (s is None and len(t) >= 0) or (t is None and len(s) >= 0) or (len(s) != len(t)):
+    if (s is None and t is not None) or (t is None and s is not None) or (len(s) != len(t)):
```

Comment:
This can be simplified to:
```suggestion
    if s is None or t is None or (len(s) != len(t)):
```
since not both `s` and `t` can be `None` (by the previous check).